stepman/paths_test.go

```diff
@@ -0,0 +1,89 @@
+package stepman
+
+import (
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"os"
+	"path"
+	"testing"
+)
+
+const (
+	GivenSteplibURI  = "https://github.com/bitrise-io/steplib"
```

Comment:
It does not really make any difference in the case of a `_test.go` file, but I would rather keep constants and `GivenRoute ` camlCase. WDYT?

Reply:
Do you mean to start with a lowercase here? like `GivenSteplibURI` ?

Reply:
yep